src/components/members.component.js

```diff
@@ -65,10 +71,14 @@ export const MembersList = ({
       showsHorizontalScrollIndicator={false}
       renderItem={({ item }) =>
         <TouchableHighlight
-          onPress={() =>
-            navigation.navigate('Profile', {
-              user: item,
-            })}
+          onPress={() => {
+            navigation.navigate(
+              authUser.login === item.login ? 'AuthProfile' : 'Profile',
```

Comment:
@RolfKoenders yep, it's good that you noticed this! I thought about this, I also did not want to duplicate the code, but where to put it? In `utils`? And how to call the function correctly, something like `getProfileScreen`?